src/App/Pages/Accounts/LockPageViewModel.cs

```diff
@@ -135,7 +135,7 @@ public string LockedVerifyText
         public async Task InitAsync()
         {
             _pinSet = await _vaultTimeoutService.IsPinLockSetAsync();
-            PinLock = (_pinSet.Item1 && _stateService.GetPinProtectedAsync() != null) || _pinSet.Item2;
+            PinLock = (_pinSet.Item1 && await _stateService.GetPinProtectedKeyAsync() != null) || _pinSet.Item2;
```

Comment:
Could we use explicit names here instead of `Item1` and `Item2`?
instead of 
`_pinSet = await _vaultTimeoutService.IsPinLockSetAsync();` 
you could have
`(isPinProtected, isPinProtectedWithKey) = await _vaultTimeoutService.IsPinLockSetAsync();`